pipeline_steps/common.groovy

```diff
@@ -92,85 +92,27 @@ def venvGalaxy(String[] args){
  *  args: list of string args to pass to ansible-playbook
  */
 def venvPlaybook(Map args){
-  withEnv(get_deploy_script_env()){
```

Comment:
This change is not specific to removing RPCO related code. `venvPlaybook` is used a lot in this repository and so I think this shouldn't be changed by this patch.

Reply:
get_deploy_script_env is only used for the old RPC-O code, because it's only related to the setting of RPC-O deploy vars - that's why this change is included.